install_build_dependencies.sh

```diff
@@ -32,6 +32,7 @@ if [ -f /etc/lsb-release ]; then
     sudo -E apt update
     sudo -E apt-get install -y \
             build-essential \
+            cmake \
```

Comment:
why? in this file below we download cmake 3.13 which is minimum required for OpenVINO build

Reply:
Why do you need to download and compile cmake when OS cmake is suitable???